display/background.py

```diff
@@ -58,6 +58,8 @@ def __init__(self, xpos, width=100):
 
         image = pygame.image.load(os.path.join("assets", "backgrounds", "city", "scroll", self.image_name))
 
+        image = pygame.transform.flip(image, bool(random.getrandbits(1)), False)
```

Comment:
This is insufficiently pythonic. The reasons for this being insufficiently pythonic are not specified, as specifying them would be non-pythonic. Instead, we recommend using duck typing to work out which reason it is.